openfisca_web_api/loader/variables.py

```diff
@@ -78,6 +78,9 @@ def build_variable(variable, country_package_metadata, tax_benefit_system):
             ),
         }
 
+    if variable.documentation:
```

Comment:
Looking at OpenFisca France, a lot of doc string is added at the _formula_ level. This PR is not exposing this doc though the API. We should probably do that too :)